"IllegalArgumentException: Unhandled parameter types" in N4JSElementSignatureProvider

```
ERROR - error during queued task: 
java.lang.IllegalArgumentException: Unhandled parameter types: [null, org.eclipse.n4js.n4JS.impl.ParameterizedPropertyAccessExpressionImpl@6fb33fbd (optionalChaining: false) (propertyAsText: null)]
	at org.eclipse.n4js.ide.server.hover.N4JSElementSignatureProvider.doGetLabel(N4JSElementSignatureProvider.java:218)
	at org.eclipse.n4js.ide.server.hover.N4JSElementSignatureProvider.get(N4JSElementSignatureProvider.java:51)
	at org.eclipse.n4js.ide.server.hover.N4JSHoverService.getContents(N4JSHoverService.java:61)
	at org.eclipse.xtext.ide.server.hover.HoverService.hover(HoverService.java:101)
	at org.eclipse.xtext.ide.server.hover.HoverService.hover(HoverService.java:65)
	at org.eclipse.n4js.ide.xtext.server.TextDocumentFrontend.hover(TextDocumentFrontend.java:288)
	at org.eclipse.n4js.ide.xtext.server.TextDocumentFrontend.lambda$4(TextDocumentFrontend.java:274)
	at org.eclipse.n4js.ide.xtext.server.ResourceTaskManager.lambda$5(ResourceTaskManager.java:247)
	at org.eclipse.n4js.ide.server.concurrent.N4JSQueuedExecutorService.lambda$0(N4JSQueuedExecutorService.java:35)
	at org.eclipse.n4js.ide.xtext.server.QueuedExecutorService$QueuedTask.run(QueuedExecutorService.java:119)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
[Error - 1:36:37 PM] Request textDocument/hover failed.
  Message: Internal error.
  Code: -32603 
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: Unhandled parameter types: [null, org.eclipse.n4js.n4JS.impl.ParameterizedPropertyAccessExpressionImpl@6fb33fbd (optionalChaining: false) (propertyAsText: null)]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at org.eclipse.n4js.ide.xtext.server.QueuedExecutorService$QueuedTask.run(QueuedExecutorService.java:132)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.IllegalArgumentException: Unhandled parameter types: [null, org.eclipse.n4js.n4JS.impl.ParameterizedPropertyAccessExpressionImpl@6fb33fbd (optionalChaining: false) (propertyAsText: null)]
	at org.eclipse.n4js.ide.server.hover.N4JSElementSignatureProvider.doGetLabel(N4JSElementSignatureProvider.java:218)
	at org.eclipse.n4js.ide.server.hover.N4JSElementSignatureProvider.get(N4JSElementSignatureProvider.java:51)
	at org.eclipse.n4js.ide.server.hover.N4JSHoverService.getContents(N4JSHoverService.java:61)
	at org.eclipse.xtext.ide.server.hover.HoverService.hover(HoverService.java:101)
	at org.eclipse.xtext.ide.server.hover.HoverService.hover(HoverService.java:65)
	at org.eclipse.n4js.ide.xtext.server.TextDocumentFrontend.hover(TextDocumentFrontend.java:288)
	at org.eclipse.n4js.ide.xtext.server.TextDocumentFrontend.lambda$4(TextDocumentFrontend.java:274)
	at org.eclipse.n4js.ide.xtext.server.ResourceTaskManager.lambda$5(ResourceTaskManager.java:247)
	at org.eclipse.n4js.ide.server.concurrent.N4JSQueuedExecutorService.lambda$0(N4JSQueuedExecutorService.java:35)
	at org.eclipse.n4js.ide.xtext.server.QueuedExecutorService$QueuedTask.run(QueuedExecutorService.java:119)
	... 5 more
```

--- a/plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/server/hover/N4JSElementSignatureProvider.xtend
+++ b/plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/server/hover/N4JSElementSignatureProvider.xtend
@@ -48,6 +48,9 @@ class N4JSElementSignatureProvider {
 			return o.typeRefAsString;
 		}
 		val id = getIdentifiableElement(o);
+		if (id === null) {
+			return null;
+		}
 		val label = doGetLabel(id, o);
 		sanitizeForHTML(label);
 		return label;

